vterm.el

```diff
@@ -868,59 +893,73 @@ in README."
 (defun vterm-previous-prompt (n)
   "Move to end of Nth previous prompt in the buffer."
   (interactive "p")
-  (if (vterm--prompt-tracking-enabled-p)
+  (if (and vterm-use-vterm-prompt
+           (vterm--prompt-tracking-enabled-p))
       (let ((pt (point)) )
         (goto-char (vterm--get-prompt-point))
         (cl-loop repeat (or n 1) do
                  (setq pt (previous-single-property-change (1- (point)) 'vterm-prompt))
                  (when pt (goto-char (1- pt)))))
     (term-previous-prompt n)))
 
+(defun vterm--get-beginning-of-line ()
+  "Find the start of the line, bypassing line wraps."
+  (save-excursion
+    (beginning-of-line)
+    (while (and (not (bobp))
```

Comment:
I haven't fully read and understood the code, but I noticed this while loop that deals with `vterm-line-warp. Is this necessary to look at every single char one at the time? Why not something like `previous-single-property-change`?

Reply:
line by line

Reply:
Oh, yes. My bad.